clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -444,10 +442,10 @@ class DockerRegistryClient {
   public void checkV2Availability() {
     try {
       doCheckV2Availability()
-    } catch (RetrofitError error) {
+    } catch (SpinnakerHttpException error) {
```

Comment:
I think this needs to be SpinnakerServerException with some additional instanceof checks for SpinnakerHttpException to get the same behavior as before.